src/helpers/data/partnerRequests.js

```diff
@@ -22,39 +22,41 @@ const getPartnerById = partnerId => new Promise((resolve, reject) => {
 });
 
 const getSinglePartner = partnerId => new Promise((resolve, reject) => {
-    axios
-        .get(`${partnersApiBaseUrl}/api/partners/${partnerId}`)
-        .then((result) => {
-            resolve(result.data)
-        })
-        .catch(error => reject(error));
-});
-
-const deletePartner = partnerId => new Promise((resolve, reject) => {
   axios
-    .delete(`${partnersApiBaseUrl}/api/partners/${partnerId}`)
+    .get(`${partnersApiBaseUrl}/api/partners/${partnerId}`)
     .then((result) => {
       resolve(result.data);
     })
     .catch(error => reject(error));
 });
 
+// const deletePartner = partnerId => new Promise((resolve, reject) => {
```

Comment:
Jonathan, looks like this Delete got commented out but I think you want to have it wrapped in a promise because the delete is working but the followup function in PartnerAdmin.js Line 228 is being run too fast so the table is not being update to reflect the delete